### PR TITLE
Rename Skymap modes; fix scene loading issue

### PR DESCRIPTION
**Pull request by Peregrine05**
- Renamed `Skymap (panoramic)` to `Skymap (equirectangular)`.
- Renamed `Skymap (spherical)` to `Skymap (angular)`.
- Fixed failure to load an angular skymap of a saved scene.
- Retain backwards-compatibility with scenes saved in older versions of Chunky.

### CODE REVIEW

File: chunky/src/java/se/llbit/chunky/renderer/scene/Sky.java [hunk 1]
```diff
@@ -681,7 +681,13 @@ public void importFromJson(JsonObject json) {
     skyExposure = json.get("skyExposure").doubleValue(skyExposure);
     skyLightModifier = json.get("skyLight").doubleValue(skyLightModifier);
     apparentSkyLightModifier = json.get("apparentSkyLight").doubleValue(apparentSkyLightModifier);
-    mode = SkyMode.get(json.get("mode").stringValue(mode.name()));
+    if (!(Objects.equals(json.get("mode").stringValue(mode.name()), "SKYMAP_PANORAMIC") || Objects.equals(json.get("mode").stringValue(mode.name()), "SKYMAP_SPHERICAL"))) {
```
Comment (leMaik) [inline chunky/src/java/se/llbit/chunky/renderer/scene/Sky.java:684]: What's the benefit of using `Objects.equals` over the `.equals` method?

File: chunky/src/java/se/llbit/chunky/renderer/scene/Sky.java [hunk 2]
```diff
@@ -699,7 +705,8 @@ public void importFromJson(JsonObject json) {
     color.set(JsonUtil.vec3FromJsonArray(json.get("color")));
 
     switch (mode) {
-      case SKYMAP_PANORAMIC: {
+      case SKYMAP_EQUIRECTANGULAR:
+      case SKYMAP_ANGULAR: {
```
Comment (leMaik) [inline chunky/src/java/se/llbit/chunky/renderer/scene/Sky.java:709]: Nice!

File: chunky/src/java/se/llbit/chunky/renderer/scene/Sky.java (hunk 1 above)
Comment (NotStirred) [reply chunky/src/java/se/llbit/chunky/renderer/scene/Sky.java:684]: Objects.equals first does a reference equality `==`, then checks for null and then does `.equals`
Though in this case they can never be `==` as (if I'm reading the brackets correctly) one is a string literal,
so `"SKYMAP_PANORAMIC".equals(json.get("mode").stringValue(mode.name()))` would probably be preferred

Comment (Peregrine05) [reply chunky/src/java/se/llbit/chunky/renderer/scene/Sky.java]: I have set it to be `json.get("mode").stringValue(mode.name()).equals("SKYMAP_PANORAMIC")`.

**Pull request merged**